cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -609,13 +632,17 @@ body {
                 margin-bottom: @grid_gutter-width * 2 / 3 ;
                 font-size: 1em;
 
+                &-heading {
+                    padding-top: 0;
+                }
+
                 // ------------------------------------------------------------
-                // TODO: Remove when Consumer Tools and Education Resources
-                //       have overview pages post-launch.
+                // TODO: Remove when Consumer Tools and Practitioner Resources
+                //       have overview pages.
                 //       This is temporary code to hide the overview links
                 //       at desktop sizes.
-                .u-link__disabled {
-                    visibility: hidden;
+                &-heading-text {
+                    .u-visually-hidden;
```

Comment:
Update to `.u-visually-hidden()` so it's clear this is a mixin and not a selector.